test/Bugs/bug_5572_wscript_loadscript_expection.js

```diff
@@ -0,0 +1,53 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+  {
+    name: "Expect exception with invalid file name to WScript.LoadScript",
+    body: function () {  
+        assert.throws(function () { 
+            WScript.LoadScript(``, ``, {
```

Comment:
Nit, is there a reason you're using format strings here rather than just standard strings?

Reply:
Absolutely no reason. Just prefer it that way :)

---
In reply to: [212401411](https://github.com/Microsoft/ChakraCore/pull/5623#discussion_r212401411) [](ancestors = 212401411)